src/components/primitives/FormattedNumber.tsx

```diff
@@ -52,11 +59,12 @@ export function FormattedNumber({
   percent,
   symbolsVariant,
   symbolsColor,
+  roundDown,
```

Comment:
Ah, the default should be here and CompactNumber should use it as a required prop.